treeherder/config/settings.py

```diff
@@ -113,11 +114,11 @@
 #
 # which django-environ converts into the Django DB settings dict format.
 DATABASES = {
-    'default': env.db_url('DATABASE_URL'),
+    'default': env.db_url('DATABASE_URL', default='http://root@mysql/treeherder'),
```

Comment:
This is a comment for my own self: Keep an eye on this differing value. This is probably `http://root@localhost:3306/treeherder`